docusaurus/docs/cloud/projects/settings.md

```diff
@@ -46,168 +48,87 @@ The ![General icon](/img/assets/icons/Faders.svg) *General* tab enables you to c
 2. In the dialog, select the reason why you are deleting your project. If selecting "Other" or "Missing feature", a textbox will appear to let you write additional information.
 3. Confirm the deletion of your project by clicking on the **Delete project** button at the bottom of the dialog.
 
-## Backups <CloudProBadge /> <CloudTeamBadge /> <UpdatedBadge /> {#backups}
+### Environments <CloudProBadge /> <CloudTeamBadge />
 
-The ![Backups icon](/img/assets/icons/ArrowClockwise.svg) *Backups* tab informs you of the status and date of the latest backup of your Strapi Cloud projects. The databases associated with all existing Strapi Cloud projects are indeed automatically backed up (weekly for Developer and Pro plans, and daily for Team plans). Backups are retained for a 28-day period. Additionally, you can create a single manual backup.
-
-:::note Notes
-
-- The backup feature is not available for Strapi Cloud projects using the free trial or the Developer plan. You will need to upgrade to either the Pro or Team plan to have your project automatically backed up and to have access to manual backups.
-
-- Only project owners can restore a backup. Maintainers have access to the ![Backups icon](/img/assets/icons/ArrowClockwise.svg) *Backups* tab but the **Restore backup** button won't be displayed for them. Refer to [Collaboration](/cloud/projects/collaboration) for more information.
-
-- The manual backup option should become available shortly after project's first succesful deployment.
-
-:::
+The ![Environments icon](/img/assets/icons/v5/Stack.svg) *Environments* tab allows to see all configured environments for the Strapi Cloud project, as well as to create new ones. Production is the default environment, which cannot be deleted. Other environments can be created (depending on the subscription plan for your project) to work more safely on isolated instances of your Strapi Cloud project (e.g. a staging environment where tests can be made before being available on production).
 
 :::tip
-For projects created before the release of the Backup feature in October 2023, the first backup will automatically be triggered with the next deployment of the project.
+Clicking on the **Manage** button for any environment will redirect you to the environment's own general settings, where it is possible to change the Node version, edit the git branches and delete or reset the environment. Please [refer to the dedicated documentation](#general-environment) for more information.
 :::
 
 <ThemedImage
-  alt="Backups"
+  alt="Project overview"
   sources={{
-    light: '/img/assets/cloud/settings_backups.png',
-    dark: '/img/assets/cloud/settings_backups_DARK.png',
+    light: '/img/assets/cloud/environments.png',
+    dark: '/img/assets/cloud/environments_DARK.png',
   }}
 />
 
-### Creating a manual backup
+To create a new environment:
```

Comment:
Yes that piece of info is written pretty clearly in the UI so I'll see later if it's worth adding it because it doesn't just impact the environments, also the plans for example. And I don't think adding more callout in that page will help with clarity. I'll think about it!